LDAPCP/LDAPCP.cs

```diff
@@ -197,7 +197,8 @@ protected bool Initialize(Uri context, string[] entityTypes)
                         this.CurrentConfiguration.LDAPConnectionsProp.Add(currentCoco.CopyPersistedProperties());
                     }
 
-                    SetCustomConfiguration(context, entityTypes);
+                    // FORTIFY WARNING: removed since this code path didn't seem to be used
+                    //SetCustomConfiguration(context, entityTypes);
```

Comment:
Please add this back, it's used by classes that inherit LDAPCP and override SetCustomConfiguration() to customize settings.